src/error.rs

```diff
@@ -34,6 +42,7 @@ impl Error {
                 message: _,
                 stderr: _,
             } => "There was an error while calling another CLI tool. Details:\n\n",
+            Error::Config { message: _ } => "There was a configuration error. Details:\n\n",
```

Comment:
we might add "crate" here, i.e. "there was a crate configuration error" - there's a lot of moving parts here so it might be useful to explain that the error was with the crate. we might want to call this CrateConfig as a result. open to ya'lls thoughts tho

Reply:
that makes sense to me, i had been considering something similar at first as well!